20180321/README.md

```diff
@@ -1 +1,188 @@
-# From 0 to Match - Set Up Everything with Apple Xcode
+# First mobile app with Matchmore
+
+In this post, we will go through setting up an iOS application with AlpsSDK, from registration on the portal to obtaining a match.
+
+## Before you start
+We will use cocoapods, to install it execute in terminal:
+```
+sudo gem install cocoapods
+```
+
+## Get API key
+Now let's create a new application on developer portal and get API key that we will later use in SDK.
+After loging in (you can register for free) you do:
+
+![alt text](https://raw.githubusercontent.com/matchmore/tech-blog/master/20180321/img/create-app.gif "create app")
+
+## Xcode mobile app
+Now we create new iOS single view application in Xcode (select swift as language) after that close Xcode and go to main application directory with terminal (in this case my-app). Execute `pod init` which will create `Podfile` file. To add AlpsSDK to application, add `pod 'AlpsSDK', '~> 0.6'` to `Podfile` 
```

Comment:
`Now let's create a fresh single view project in Xcode (I highly recommend using Swift🤓)`